e2e/cypress/integration/enterprise/system_console/channel_moderation/manage_members_spec.js

```diff
@@ -56,7 +56,7 @@ describe('MM-23102 - Channel Moderation - Manage Members', () => {
         });
     });
 
-    it('No option to Manage Members for Guests', () => {
+    it('MM-T1547 No option to Manage Members for Guests', () => {
```

Comment:
This has 2 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

Reply:
Consolidated steps in TM4J